activestorage/app/models/active_storage/blob.rb

```diff
@@ -27,9 +27,7 @@ class ActiveStorage::Blob < ActiveRecord::Base
 
   self.table_name = "active_storage_blobs"
 
-  MINIMUM_TOKEN_LENGTH = 28
```

Comment:
Should we keep this as a constant? We do use it in two places.

Reply:
👍